<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HttpAnalyse</class>
 <widget class="QDialog" name="HttpAnalyse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>点击获取</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置抓包分析URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PS：&quot;点击获取&quot; 后请及时访问或刷新 包含URL的页面</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>148</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>196</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>238</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>326</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainedit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainedit1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>151</y>
     <width>130</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Accept:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Content-Type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Referer:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Accept-Language:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Accept-Encoding:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User-Agent:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cookies：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PostString：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
